public/js/main.js

```diff
@@ -288,10 +289,17 @@ function setFileURLs(new_provider) {
       appLoading.stop();
     } else if (lastQuery !== val) {
       animateTop();
-      index.search(val, displayMatchingLibraries);
+      displayItems = 20;
+      index.search(val, {hitsPerPage: displayItems}, displayMatchingLibraries);
     }
     lastQuery = val;
   }
+  $(window).scroll( () => {
```

Comment:
I wonder if it's the time we'll start to use syntax like `() => {}` since not everyone knows about it and it'll definitely increase the gap for the new contributors to get started on contribution, maybe should rollback this part before we've made a decision to migrate to the usage.